build.sbt

```diff
@@ -39,18 +42,18 @@ def commonProject(id: String): Project = {
 
 def sttpPlayJson(includePlayVersion: String, scalaVersion: String): Project = {
   val (playSuffix, crossScalaVersion) = includePlayVersion match {
-    case Dependencies.play23Version => ("23", CrossScalaVersion)
-    case Dependencies.play25Version => ("25", CrossScalaVersion)
-    case Dependencies.play26Version => ("26", Seq.empty)
+    case Dependencies.play23Version => ("23", Seq.empty)
+    case Dependencies.play25Version => ("25", Seq.empty)
+    case Dependencies.play26Version => ("26", CrossScalaVersion)
   }
 
   val id = s"sttp-play$playSuffix-json"
-  commonProject(id).settings(
-    sourceDirectory := baseDirectory.value / "sttp-play23-json" / "src",
```

Comment:
sbt should be able to resolve this

Reply:
The point of this was so that we only needed the source files in a single directory. By doing this now we need to copy the source for each version. I think we should only do that if the implementations diverge.

Reply:
Thats a good idea, didn't notice that.
It doesn't seem to be able to pick up the file in the current master.